src/include/rbd/librbd.h

```diff
@@ -569,6 +569,10 @@ CEPH_RBD_API int rbd_get_features(rbd_image_t image, uint64_t *features);
 CEPH_RBD_API int rbd_update_features(rbd_image_t image, uint64_t features,
                                      uint8_t enabled);
 CEPH_RBD_API int rbd_get_op_features(rbd_image_t image, uint64_t *op_features);
+CEPH_RBD_API int rbd_features_to_string(rbd_image_t image, uint64_t *features,
+                                        char *str_features);
```

Comment:
Nit: you need a max length in/out parameter and then you need to ensure it has enough space to hold the resulting string.